src/libraries/System.Text.RegularExpressions/src/Resources/Strings.resx

```diff
@@ -57,10 +57,13 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="AlternationCantCapture" xml:space="preserve">
+  <data name="Unknown" xml:space="preserve">
```

Comment:
It's confusing that this is named the same as the enum member, but unlike the other strings, it's not the text for that enum member.

Reply:
@stephentoub I'm not sure I follow, matching the enums in the resources was a specific request, and matches how it was. If you think the text can be improved, please do, I'm on mobile and can't make the changes. Note that the `Unknown` is not used internally, atm.

Reply:
This resource isn't at all related to the enum Unknown member.

Reply:
@stephentoub, it says: "regular expression parser error at offset", that's precisely what it should say. What's your suggestion?

Reply:
Please stop resolving comments that aren't resolved.

My point is that all the other relevant resources are named after the enum value that they're then used with. That's not the case with this one. So it's not the text in complaining about, it's the name of the resource.

Reply:
How about SR.Generic?